developer/config.yaml

```diff
@@ -6,10 +6,10 @@
 cluster_type: openshift
 
 # git_url refers to a git repo to be considered as the source of truth for Argo CD applications.
-git_url: https://github.com/openshift-pipelines/pipeline-service.git
+git_url: https://github.com/AndrienkoAleksandr/pipeline-service.git
```

Comment:
Revert changes to this file. To avoid having to modify that file and commit it by mistake, you can use the `--use-current-branch` when running `dev_setup.sh`.